src/messages/copy.rs

```diff
@@ -60,8 +58,7 @@ impl Message for CopyDone {
 
 pub const MESSAGE_TYPE_BYTE_COPY_FAIL: u8 = b'f';
 
-#[derive(Getters, Setters, MutGetters, PartialEq, Eq, Debug, Default, new)]
-#[getset(get = "pub", set = "pub", get_mut = "pub")]
+#[derive(PartialEq, Eq, Debug, Default, new)]
 pub struct CopyFail {
     message: String,
```

Comment:
Do we need to make members of all message structs `pub` for consistency?

Reply:
Yes, I missed this one